.github/workflows/markdown.yaml

```diff
@@ -92,3 +96,14 @@ jobs:
         config-file: '.github/config/md-link-config.json'
         use-verbose-mode: 'yes'
         file-extension: '.MD'
+  markdown-spellchecker:
+    if: ${{ (github.event_name == ' workflow_call' && inputs.enable-spell-check) || github.event_name != ' workflow_call' }}
```

Comment:
Is the whitespace in `' workflow_call'` intentional?

Reply:
of course not :) good catch!